### PR TITLE
:sparkles: Add `MultipleHubs` in the feature gate list.

### PR DESCRIPTION
**Pull request by xuezhaojun**
## Summary

The `MultipleHubs` is in alpha stage, is used by both operator and registration-agent.

### CODE REVIEW

File: feature/feature.go [hunk 1]
```diff
@@ -74,6 +74,9 @@ const (
 
 	// ResourceCleanup will start gc controller to clean up resources in cluster ns after cluster is deleted.
 	ResourceCleanup featuregate.Feature = "ResourceCleanup"
+
+	// SwitchHub allows user to configure multiple bootstrapkubeconfig via Klusterlet and let agent to decide which one to use
+	SwitchHub featuregate.Feature = "SwitchHub"
```
Comment (qiujian16) [inline feature/feature.go]: MultipleHubs might be a better name. Thought @elgnay ?

Comment (xuezhaojun) [reply feature/feature.go]: Agree, the `MultipleHubs` is better, fixed.

File: feature/feature.go [hunk 2]
```diff
@@ -74,6 +74,9 @@ const (
 
 	// ResourceCleanup will start gc controller to clean up resources in cluster ns after cluster is deleted.
 	ResourceCleanup featuregate.Feature = "ResourceCleanup"
+
+	// MultipleHubs allows user to configure multiple bootstrapkubeconfig via Klusterlet and let agent to decide which one to use
```
Comment (qiujian16) [inline feature/feature.go]:
```suggestion
	// MultipleHubs allows user to configure multiple bootstrapkubeconfig connecting to different hubs via Klusterlet and let agent decide which one to use
```

Comment (xuezhaojun) [reply feature/feature.go]: Updated.

**Pull request merged**